src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -335,23 +335,17 @@ network::network(cldnn::BinaryInputBuffer& ib, stream::ptr stream, engine& engin
     , _reset_arguments(true) {
     net_id += 1;
 
-    uint32_t prog_id;
-    std::vector<std::string> batch_header_str;
-    ib >> prog_id;
-    ib >> batch_header_str;
-    kernels_cache kernels_cache(get_engine(), prog_id, batch_header_str);
+    kernels_cache kernels_cache(get_engine(), 0, {""});
     ib >> kernels_cache;
 
     int num_data_nodes;
     ib >> num_data_nodes;
 
-    _memory_pool->clear_pool_for_network(net_id);
```

Comment:
https://github.com/openvinotoolkit/openvino/pull/13801#discussion_r1021125100
I removed it. Thank you.